### ISSUE TITLE
Code preview not including same headers as postman call

### ISSUE THREAD

**Issue by robinglen** [a85/POSTMan-Chrome-Extension#3333]
I'm not sure this is a bug but can get confusing.

When using postman if I make a GET request to an API it appears to add accept-encoding headers under the hood because for example if I call the following:
`http://www.matchesfashion.com/mens/just-in/just-in-this-month?page=1&noOfRecordsPerPage=60&sort=&q=&format=json&navMode=notfull&noattraqt=Set`

You will see it returns the headers for gzip encoding.

However if I then use the request code snippet generated:

```Javascript
var request = require("request");

var options = { method: 'GET',
  url: 'http://www.matchesfashion.com/mens/just-in/just-in-this-month',
  qs: 
   { page: '1',
     noOfRecordsPerPage: '60',
     sort: '',
     q: '',
     format: 'json',
     navMode: 'notfull',
     noattraqt: 'Set' },
  headers: 
   { 'postman-token': '37a63ec0-a4da-795d-4c51-940b0eb1526c',
     'cache-control': 'no-cache' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
```
This response is not gzipped, if you add the option: `resolveWithFullResponse: true` you could see this.

To get the same response you need to add the option, `gzip: true`.

I found it a little confusing the postman doesn't say that its adding the headers for `accept-encoding`.

Thoughts?

**Comment by sdnts** [comment]
Hi @thearegee, are you on the native apps?
These are added by the library we use to actually send the request. I'll make it so the generated code snippets match. Thanks for letting me know.

**Comment by robinglen** [comment]
Hey I'm using the Chrome App - no problem, messed my head right out last night finding this issue :)

**Comment by abhijitkane** [comment]
@shamasis Some sort of preview from the runtime could help with cases like this.

**Comment by abhijitkane** [comment]
@thearegee Apologies for the inordinate delay here. In Postman 7.2.1, we've introduced bug-fixes, support for two new languages (JS-Fetch and Powershell), and the ability to configure certain aspects of the generated snippet with the 'Experimental Codegen Mode' flag enabled in Postman's global settings. One of the code-specific settings is the ability to `Include Temporary Headers` - these are headers added by the Postman app before sending the request.

<img width="785" alt="Screen Shot 2019-06-19 at 13 14 15" src="https://user-images.githubusercontent.com/681190/59746266-2aa85200-9294-11e9-91dd-6b257b2afa7e.png">

Closing this for now. Let me know if you face any other issues around this.
